system-tests/test/downloads_spec.ts

```diff
@@ -23,7 +23,7 @@ describe('e2e downloads', () => {
 
   systemTests.it('allows changing the downloads folder', {
     project: 'downloads',
-    spec: '*',
+    spec: 'downloads.cy.ts',
```

Comment:
@ryanthemanuel fixed some flake with this spec in develop and had changed `*` to `downloads_spec.ts`. However, `downloads_spec.ts` caused the failure in CI in this PR because we renamed the file to `downloads.cy.ts` in 10.0